sdk/src/main/java/ly/count/android/sdk/ModuleConfiguration.java

```diff
@@ -26,6 +26,7 @@ class ModuleConfiguration extends ModuleBase implements ConfigurationProvider {
 
     boolean currentVTracking = true;
     boolean currentVNetworking = true;
+    boolean configurationFetched = false;
```

Comment:
this serves as a "was this method called before". what was the necessity? I vaguely remember we fetched things twice (or more?)? was it not?

Reply:
I added this because I needed to add fetching after exiting from the temp id mode. 
When user enables temp id in the initialization, those calls should not happen. However, we need to call them after user exits from it. 

And I feel like it is ok to only fetch configuration once and sending health check once. K also said we should send/fetch them only once